demo.py

```diff
@@ -58,7 +65,7 @@ def train_model(X_train, y_train, X_test, y_test, lmd):
     N_test = len(X_test)
 
     print("\n======================================")
-    print("Train#: %d, Test: %d" %(N_train, N_test))
+    print("Train: {}, Test: {}".format(N_train, N_test))
```

Comment:
Good suggestion.  I have added a short description of the data set to the start of the output, and as discussed separately the demo has been updated so that only one data set can be run at a time.